nightwatch.conf.js

```diff
@@ -70,7 +70,7 @@ module.exports = {
       }
     },
     saucelabs: {
-      selenium_host: 'saucelabs',
+      selenium_host: SELENIUM_HOST,
```

Comment:
This is already in the default, so, it's no longer needed here.
```suggestion
```

Reply:
I don't see any change for this file now in the diff. Isn't this PR all about this change?
To clarify, I simply suggested to remove the specific line, and after that, setting SELENIUM_HOST should work as it's on the default.

Reply:
:man_facepalming:

Reply:
Sorry, I could not be clearer. What i said is you no longer need that line. You can remove line 73, and it should work as it's already present in the default.